cluster/gce/config-default.sh

```diff
@@ -41,7 +41,8 @@ MASTER_TAG="${INSTANCE_PREFIX}-master"
 MINION_TAG="${INSTANCE_PREFIX}-minion"
 MASTER_IP_RANGE="${MASTER_IP_RANGE:-10.246.0.0/24}"
 CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-10.244.0.0/16}"
-MINION_SCOPES=("storage-ro" "compute-rw" "https://www.googleapis.com/auth/monitoring" "https://www.googleapis.com/auth/logging.write")
+MINION_SCOPES="${MINION_SCOPES:-compute-rw,monitoring,logging-write,storage-ro}"
```

Comment:
Based on prior experience, I think that help text may be incorrect. We tried `logging-write` in https://github.com/GoogleCloudPlatform/kubernetes/pull/7324 and had to fix it to `logging.write` in https://github.com/GoogleCloudPlatform/kubernetes/pull/7458. The help text for creating GCE instances shows the same output, so I think the help text is wrong.